ethexe/common/src/lib.rs

```diff
@@ -20,54 +20,23 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(feature = "std")]
```

Comment:
is it needed somewhere outside of tests?

Reply:
This can be used in outside std crates

Reply:
so what? `#![cfg_attr(not(feature = "std"), no_std)]` does exactly the same

Reply:
Show me code ssuggestion. I cannot understand your oint

Reply:
removed